diff --git a/node_modules/puppeteer/lib/types.d.ts b/node_modules/puppeteer/lib/types.d.ts
index 42c120c..39a858a 100644
--- a/node_modules/puppeteer/lib/types.d.ts
+++ b/node_modules/puppeteer/lib/types.d.ts
@@ -24382,4 +24382,4 @@ export declare type WrapElementHandle<X> = X extends Element ? ElementHandle<X>
 export { }
 
 declare const puppeteer: PuppeteerNode;
-export = puppeteer;
\ No newline at end of file
+export default puppeteer;
\ No newline at end of file
